app/src/main/res/layout/content_main.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I dont really know what it does, i know its similar to a relative layout but again it was there when i got here